src/screens/Gallery/gallery.css

```diff
@@ -0,0 +1,94 @@
+/* -----------------------------------------------VARIABLES--------------------------------------------------------------- */
+:root {
+	--white: #ffffff;
+	--black: #000000;
+	--center: center;
+	--backgroundRepeat: no-repeat;
+	--fontFamily: 'Poppins', sans-serif;
+}
+
+.classContainer {
+	width: 90vw;
+	height: 70vh;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-around;
+	align-items: stretch;
+	align-content: stretch;
+	margin: auto;
+}
+.card {
+	margin: auto;
+	box-shadow: 0px 4px 4px var(--black);
+	transition: 0.3s ease-in;
+	background-color: #324359 !important;
```

Comment:
any reason why you're using !important here?  Are you trying to override some other styling?

Reply:
at first i had a diff style so yea i tried to overwrite than i never delete it